packages/calypso-e2e/src/lib/components/navbar-component.ts

```diff
@@ -71,20 +59,17 @@ export class NavbarComponent {
 	async openNotificationsPanel( {
 		useKeyboard = false,
 	}: { useKeyboard?: boolean } = {} ): Promise< void > {
-		await this.pageSettled();
-
-		const notificationsButton = await this.page.waitForSelector( selectors.notificationsButton, {
-			state: 'visible',
-		} );
 		await Promise.all( [
 			this.page.waitForLoadState( 'networkidle' ),
-			notificationsButton.waitForElementState( 'stable' ),
+			this.page.waitForResponse( /async-load-calypso-blocks-jitm.*/, { timeout: 15 * 1000 } ),
```

Comment:
@WunderBart Do you think waiting on this is risky? 

I am trying to ensure that Calypso has fully loaded before attempting to interact with the NavBar > Notification button. Otherwise, in some cases the click can be swallowed up by Calypso fully loading. 

I don't really have a better request or element to key off other than this - the other, safer fallback would be `waitForLoadState(networkidle)`. What do you think?

Reply:
From my perspective, I'm okay with this for now! Out of curiosity, did it not work to wait for another element on the navbar to ensure it's stable?

Reply:
@dpasque it was not sufficient to wait on other elements on the navbar. 

This can be observed by placing a breakpoint on the XHR > Load event then loading `/home`. What happens is:

The initial load occurs
![image](https://user-images.githubusercontent.com/6549265/205174718-7d2bb41f-3651-4fed-a249-4bd62bd64f61.png)

The navbar loads all at once
![image](https://user-images.githubusercontent.com/6549265/205174744-3e67c5af-0362-4a60-9a1e-60a1b6d145cc.png)

The SidebarComponent then begins the loading process
![image](https://user-images.githubusercontent.com/6549265/205174823-bee2ce8f-1ed1-44d1-9a6f-e37b5c2a54de.png)

(and then many XHR requests later)
The SidebarComponent is mostly loaded and the home page cards being loading
![image](https://user-images.githubusercontent.com/6549265/205174961-c83aed1e-d8d4-40c5-aa13-415b0b97c880.png)

Since the Navbar loads all at once, waiting on other elements contained within the Navbar is insufficient to guarantee that the page is loaded.

Reply:
Makes sense -- thanks for explaining!

Reply:
Thanks @WunderBart, for always offering up words of wisdom.

> If the request is not directly related to the action element, it should be extracted from the action method and handled independently (e.g., at the page-load time), or

I think this will be the path I take to address this issue. The navbar should not have to be aware of the load state of the rest of the page; that's a job for either the spec to ensure, or another POM (eg. `MyHomePage`).

Reply:
Filed https://github.com/Automattic/wp-calypso/issues/70961 to make required changes.